docs/src/docs/asciidoc/index.adoc

```diff
@@ -1,496 +1,37 @@
-    </pluginRepository>
-</pluginRepositories>
-----
-
-=== Release 0.9.3
-
-This release contains:
-
-- Fix for mainClass not being optional (Gradle plugin)
-- Fix for Gradle < 7 failing to determine GraalVM toolchain
-- Gradle plugin now registers proper groups
-- Automatic native-image tool fetching via gu (Gradle plugin)
-- FIxed issue where nativeTest would fail when tests are annotated with Timeout
-- Added a sharedLibrary configuration option for Gradle plugin
-- Removed broken server configuration option from Gradle plugin
-- Added a documentation website with proper CI integration
-
-In addition to those improvements, several behind-the-scenes changes were made:
-
-- Introduced "Dockerless" Maven plugin functional testing
-- Parallelized Gradle testing in CI
-- Replaced groovy-json with jackson-databind for JSON handling
-- Fixed Github Actions syntax to enable manual workflow invoking
-
-=== Release 0.9.2
-
-This release contains:
-
-- Revamped Gradle plugin that is now a lot more idiomatic.
-- Fixes for several issues regarding JUnit testing.
-- Removal of Test Discovery mode from the Maven plugin.
-- Fix for Maven creating empty test images when no tests are present.
-- Added support for Kotlin tests in Gradle.
+[discrete]
+=== Documentation
 
-In addition to those improvements, several behind-the-scenes changes were made in order to ensure better compatibility moving forward:
+In case you are new to the Native Build Tools (either as an end user or a library author), read our:
 
-- Test coverage has been greatly improved for all subprojects.
-- Build tooling for this repository has been improved significantly.
+- <<end-to-end-gradle-guide.adoc#,End-to-end Gradle guide>>
+- <<end-to-end-maven-guide.adoc#,End-to-end Maven guide>>
 
-Note that there has been a breaking change in the Gradle plugin - `persistConfig` configuration option was removed.
-Using said option will cause existing builds to break, so users are advised to remove it from their configuration prior to upgrading.
-System property `-DpersistConfig` will have no effect going forward.
+For detailed documentation, please refer to the following pages of the specific build tools:
 
-=== Release 0.9.1
+- <<gradle-plugin.adoc#,The Gradle plugin documentation>>
+- <<maven-plugin.adoc#,The Maven plugin documentation>>
 
-This release contains:
+You can see the full changelog <<changelog.adoc#,here>>.
 
-- Fixes for most of the known issues regarding Gradle and Maven plugins
-- Massively improved automatic JUnit support as well as initial JUnit Vintage support
-- Improved JavaDoc and tests for the Gradle plugin
+[discrete]
+=== Contributing
 
-=== Release 0.9.0
+If you are interested in contributing or reporting an issue related to the plugins, please refer to the https://github.com/graalvm/native-build-tools[Native Build Tools repository].
+Also, be aware that all contributors must sign the https://oca.opensource.oracle.com/[Oracle Contributor Agreement] (OCA).
```

Comment:
```suggestion
```suggestion
Please note that all contributors must sign the https://oca.opensource.oracle.com/[Oracle Contributor Agreement] (OCA).
```